CMake/Modules/FindLibusb.cmake

```diff
@@ -39,12 +39,11 @@ if ( PKGCONFIG_LIBUSB_FOUND )
 else ( PKGCONFIG_LIBUSB_FOUND )
   find_path ( LIBUSB_INCLUDE_DIRS
     NAMES
-      usb.h
+      libusb.h
```

Comment:
I think this CMake file need complete refactoring.
Because at line 22 we're trying to find libusb. but must be libusb-1.0.

So, technically this files does:
1. using pkgconfig system detects library (and its include/link flags). 
2. if library was not found you're trying to ring libusb.h at different places, like programfiles and user-never-defined libUSB_ROOT_DIR, and as last resort - git submbodule.
3. do some platform specific stuff, like adjust libraries path, to link to
4. else find library libusb in general (for linking)
5. checks again for libusb.h
6. fails if not found

And this logic is broken: if libusb (0.x) is found then go with it else link libusb (1.x) from extern/libusb, then link libusb (0.x) again and use libusb (0.x)